src/main/webapp/jsp/instructorFeedbackResultsByRecipientGiverQuestion.jsp

```diff
@@ -251,7 +251,7 @@
                                     <span class='glyphicon glyphicon-chevron-up pull-right'></span>
                                 </div>
                                 <div class='panel-collapse collapse in'>
-                                    <div class="panel-body"> There are no responses received by this user 
+                                    <div class="panel-body"> <i>There are no responses given by this user</i>
```

Comment:
Why change from "received" to "given"? The panels are displayed by recipient on this page.